### DISCUSSION TITLE
Change theme based on windows dark/light mode

### DISCUSSION THREAD

**Discussion by user9931** [TriliumNext/Trilium#2906] [Q&A]
yes, i understand that it's a very strange thing that maybe noone else want... 
Is it possible to change the scheme based on windows dark / light mode?

**Comment by zadam** [comment]
This is not supported at the moment.

Trilium tries to detect the color scheme (light/dark) at the initialization, but this is afterwards not re-checked.

If you really want this and have JS scripting skills, this could be hacked together as a custom frontend script.
